GitUI/UserControls/FileStatusList.cs

```diff
@@ -72,8 +72,11 @@ public FileStatusList()
             FileStatusListView.SmallImageList = _images;
             FileStatusListView.LargeImageList = _images;
 
-            NoFiles.Visible = false;
+            HandleVisibility_NoFilesLabel_FilterComboBox(filesPresent: true);
+            this.Controls.SetChildIndex(NoFiles, 0);
             NoFiles.Font = new Font(SystemFonts.MessageBoxFont, FontStyle.Italic);
+
+            _filter = new Regex(".*");
```

Comment:
I would assign a null value when the filter value is empty. The condition for accepting files should accept all files when the `_filter` is null.